storage/mroonga/vendor/groonga/lib/ii.c

```diff
@@ -3338,6 +3342,8 @@ fake_map(grn_ctx *ctx, grn_io *io, grn_io_win *iw, void *addr, uint32_t seg, uin
   iw->addr = addr;
 }
 
+PRAGMA_DISABLE_CHECK_STACK_FRAME
+
```

Comment:
monty pushed the disables of this and below in f65dda628d5cf75dec31530a4fab94cc139b45c3

So only above chunk for `buffer_segment_reserve` is clang specific.

Reply:
There seems to be a desync between you and the others that you (plural) were doing overlapped work, complete with (specifically the others) skipping Jira logs.

Reply:
resolved necessary code conflicts.